bin/find-spark-home.cmd

```diff
@@ -0,0 +1,49 @@
+@echo off
+
+rem
+rem Licensed to the Apache Software Foundation (ASF) under one or more
+rem contributor license agreements.  See the NOTICE file distributed with
+rem this work for additional information regarding copyright ownership.
+rem The ASF licenses this file to You under the Apache License, Version 2.0
+rem (the "License"); you may not use this file except in compliance with
+rem the License.  You may obtain a copy of the License at
+rem
+rem    http://www.apache.org/licenses/LICENSE-2.0
+rem
+rem Unless required by applicable law or agreed to in writing, software
+rem distributed under the License is distributed on an "AS IS" BASIS,
+rem WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+rem See the License for the specific language governing permissions and
+rem limitations under the License.
+rem
+
+rem Path to Python script finding SPARK_HOME
+set FIND_SPARK_HOME_PYTHON_SCRIPT=%~dp0find_spark_home.py
```

Comment:
I think this fails rather uglily if python is not in the path? should we add some checks for that?

Reply:
I manually tested and it looks going to give a message like this:

```cmd
C:\...>pyspark
'python' is not recognized as an internal or external command, operable program or batch file.
```

which seems roughly fine though it's bit uglily. 

So, googled a possible approach, for example, https://superuser.com/a/718194. However, seems `where` does not recognise an absolute path, for example, `C:\...\Python27\python.exe`. So, looks we should make a combination with `exists` keyword.

@jsnowacki if you are active now and know a simple better way, we could definitely try. Or, probably, we could go ahead as is too ..

Reply:
The only idea I have at the moment is adding is something along the line:

```cmd
rem If there is python installed, try to use the root dir as SPARK_HOME
where %PYTHON_RUNNER% -c"print('hi!')"
if %ERRORLEVEL% neq 0 (
	echo %PYTHON_RUNNER% wasn't found 
	if "x%SPARK_HOME%"=="x" (
		set SPARK_HOME=%~dp0..
	)
)
```

I need to add it though in a main code not in the conditional subpblock as it doesn't work inside a block of code due to this CMD limitations regarding env variable evaluation.